healthcare/api-client/v1/consent/attribute_definitions_test.py

```diff
@@ -0,0 +1,210 @@
+# Copyright 2022 Google LLC
```

Comment:
Any test cases for `patch_attribute_definition`?

Reply:
Done.